helpers/GenerateResourcesAndImage.ps1

```diff
@@ -171,27 +174,28 @@ Function GenerateResourcesAndImage {
             $options = [System.Management.Automation.Host.ChoiceDescription[]]($yes, $no, $stop)
             $result = $host.ui.PromptForChoice($title, $message, $options, 0)
 
-            switch ($result)
-            {
+            switch ($result) {
                 0 { Remove-AzResourceGroup -Name $ResourceGroupName -Force; New-AzResourceGroup -Name $ResourceGroupName -Location $AzureLocation }
                 1 { <# Do nothing #> }
                 2 { exit }
             }
         }
-    } else {
+    }
+    else {
         New-AzResourceGroup -Name $ResourceGroupName -Location $AzureLocation
     }
 
     # This script should follow the recommended naming conventions for azure resources
-    $storageAccountName = if($ResourceGroupName.EndsWith("-rg")) {
-        $ResourceGroupName.Substring(0, $ResourceGroupName.Length -3)
-    } else { $ResourceGroupName }
+    $storageAccountName = if ($ResourceGroupName.EndsWith("-rg")) {
```

Comment:
Lets replace to:
```
$storageAccountName = $ResourceGroupName
if ($ResourceGroupName.EndsWith("-rg")) {
    $storageAccountName = $ResourceGroupName.Substring(0, $ResourceGroupName.Length - 3)
}
```

Reply:
I didn't change theses lines, it's a plugin in vscode who automatic format. I'll revert it